spec/factories.rb

```diff
@@ -0,0 +1,21 @@
+FactoryGirl.define do
+
+  factory :user do |u|
+    sequence(:email) { |n| "example#{n}@yahoo.com" }
+    u.password { 'password' }
+    u.password_confirmation { 'password' }
+  end
+
+  factory :vault do |v|
+    v.name { "Vault" }
+    v.exposure { rand(0..2) }
+    v.user_id { rand(1..10) }
+  end
+
+  factory :snippet do |s|
+    s.name { 'Snippet' }
+    s.language { rand(0..30) }
+    s.code { 'Test test test test' }
+    s.vault_id { rand(1..100) }
```

Comment:
Before start working on new features I want to setup a community on Slack or something similar to enhance communication between contributors. I will also create a website to promote and explain my idea. Thank you for all your help!